src/components/counter-element.js

```diff
@@ -25,9 +25,8 @@ class CounterElement extends LitElement {
       <div>
         <p>
           Clicked: <span>${props.clicks}</span> times.
-          Value is <span>${props.value}</span>.
-          <button on-click="${() => this._onIncrement()}" title="Add 1">${plusIcon}</button>
           <button on-click="${() => this._onDecrement()}" title="Minus 1">${minusIcon}</button>
+          <button on-click="${() => this._onIncrement()}" title="Add 1">${plusIcon}</button>
```

Comment:
I don't think it's necessary to change this ordering, and it's consistent with the original redux example: https://github.com/reduxjs/redux/blob/master/examples/counter-vanilla/index.html#L11

Reply:
Didn't know about that, makes sense.